scrabble.js

```diff
@@ -1,16 +1,136 @@
 
+
+
 const Scrabble = {
   score(word) {
+    if(word === "" || word == null) {
+      throw new Error ("Must enter a word with at least one character")
+    }
```

Comment:
I like that you're throwing full instances of `Error` here. It's tempting in JS to throw raw strings, but then you don't get a stack trace.